cmd/nodecmd/ssh.go

```diff
@@ -69,12 +74,31 @@ func sshNode(_ *cobra.Command, args []string) error {
 				if err != nil {
 					return err
 				}
+				monitoringInventoryPath := filepath.Join(app.GetAnsibleInventoryDirPath(clusterNameOrNodeID), constants.MonitoringDir)
```

Comment:
we should not allow running ssh commands for the cluster on monitoring instance as it's very different.  Everything else is the same so it makes sense to allow it as all boxes are deployed exactly the same so there is value in running one command across cluster.
For monitoring intance as it's not running avalanchego - it should be excluded from cluster wide ssh commands.

But I still see value allowing users to reference monitoring instance per cloud instance ID to run ssh command or open terminal sessions